test/run.sh

```diff
@@ -252,7 +249,13 @@ do
     # extra flags (allow shell variables there)
     moc_extra_flags="$(eval echo $(grep '//MOC-FLAG' $base.mo | cut -c11- | paste -sd' '))"
     moc_extra_env="$(eval echo $(grep '//MOC-ENV' $base.mo | cut -c10- | paste -sd' '))"
-    moc_with_flags="env $moc_extra_env moc $moc_extra_flags $EXTRA_MOC_ARGS"
+    if ! grep -q "//MOC-NO-FORCE-GC" $base.mo
```

Comment:
Would it be easier to just add a `--dont-force-fc flag` to moc for those few test?

Reply:
I was considering so, but the problem was that `drun-wrapper.sh` always unconditionally set `--force-gc` and combining this with an additional `--dont-force-gc` would be contradicting `moc` argument. 

In `drun-wrapper.sh`:
```
EXTRA_MOC_ARGS="--force-gc $EXTRA_MOC_ARGS"
```

Maybe there is a nicer solution that we conditionally activate `force-gc` without having to adjust the other tests...

Reply:
Right, I'd be fine with it cancelling the preceding --force-gc.

Or // SKIP force-gc? to opt out?

Reply:
Maybe, cancelling `--force-gc` would be the nicest solution. And activating it in the critical test cases that test the GC. I will adjust this.